src/content/components/FlameChartViewport.js

```diff
@@ -1,9 +1,26 @@
-import React, { Component, PropTypes } from 'react';
+// @flow
+import React, { Component } from 'react';
 import FlameChartCanvas from './FlameChartCanvas';
+import type { Thread } from '../../common/types/profile';
+import type { Milliseconds, CssPixels, UnitIntervalOfProfileRange } from '../../common/types/units';
+import type { StackTimingByDepth } from '../stack-timing';
+
+type Props = {
+  connectedProps: {
```

Comment:
Nevermind. Removed it and enumerated everything instead. Looks like I can't decide what I'm doing.